pgLoad.dhall

```diff
@@ -4,7 +4,7 @@
   ''
   LOAD database
   FROM '${sqliteFile}'
-  INTO ${pgUri}
+  INTO ${pgUri}?sslmode=require
```

Comment:
```suggestion
  INTO ${pgUri}?sslmode=accept
```

Maybe worth trying this out to see it if also works… could be more amenable to local testing.